anagram_program.c

```diff
@@ -0,0 +1,64 @@
+#include <stdio.h>
```

Comment:
Add program description at the top and update README.md to get this PR merged.

Reply:
I have added the program description however I am a bit confused about how to edit the README.md file. Could you please guide me.